interlok-azure-datalake/src/main/java/com/adaptris/interlok/azure/datalake/DataLakeProducer.java

```diff
@@ -100,6 +102,14 @@ protected void doProduce(AdaptrisMessage adaptrisMessage, String endpoint) throw
       DataLakeConnection connection = retrieveConnection(DataLakeConnection.class);
       DataLakeServiceClient dataLakeServiceClient = connection.getClientConnection();
       DataLakeFileSystemClient fileSystemClient = dataLakeServiceClient.getFileSystemClient(d);
+      String hierarchy = "";
+      for (String dir : p.split("/")) {
```

Comment:
Not knowing anything about Datalake; can we assume a file path separator always being "/"?
Probably right?

Reply:
I hope so - I was basing my decision on their documentation https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-get-started-java-sdk which uses / in the paths. I couldn't find anything in their `Constants` class, in addition their `DataLakePathClientBuilder` class uses `String.format("%s/%s/%s" ... `to build paths.